applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>I think we should consider putting a citation for the Magnus-Tetens Approximation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>